interface/main/main_screen.php

```diff
@@ -437,6 +493,24 @@ function generate_html_end()
     exit();
 }
 
+?>
+<?php
+$attr_url = 'attr_url';
+$text = 'text';
+$xlt = 'xlt';
+$attr = 'attr';
+
+    $totp_top = <<<TPT
+    <div class="container">
+        <div class="row">
+            <div class="col-sm-12">
+                <div class="page-header">
+                    <h2>{$xlt('TOTP Verification')}</h2>
+                </div>
+            </div>
+        </div>
+TPT;
+    echo $totp_top ."\r\n";
```

Comment:
Is this type of code block needed (this script is rather confusing already)?

Reply:
sorry i uploaded the wrong file

Reply:
give me  a sec I will post the correct one

Reply:
no prob. gonna look nice :)

Reply:
See if the logic works  with U2F

Reply:
Is the above code block supposed to be there?